### ISSUE TITLE
Handle multiple-frame scenes (visualization)

### ISSUE THREAD

**Issue by chris-jenkins-isi** [isi-vista/adam#579]
Depends on #572 for the basis of the data that needs to be visualized. Poses some UI questions for how best to indicate whether one is looking at the before or after action view of the scene.

**Comment by chris-jenkins-isi** [comment]
There's also the consideration of scenes in which it's significant what occurs in the middle (e.g. with throwing, it's important that the thrown object is in the air before it hits the ground).

**Comment by gabbard** [comment]
For now, just work with two frames (before/after) and kick dealing with what happens in the middle to another issue.

**Comment by chris-jenkins-isi** [comment]
To accomplish this, the frame information needs to be included in the call to `viz.set_title()` within `make_scenes.py` -- as suggested by the TODO in `SceneCreator.create_scenes()`, the current frame and total number of frames can be added to the `SceneElements` class.

**Comment by chris-jenkins-isi** [comment]
Closed via #626
